public/content/lessons/2017/derivatives-trig-functions/index.mdx

```diff
@@ -0,0 +1,62 @@
+
+For a given value of theta, like $0.8$, you’d imagine walking around the circle, starting from the rightmost point, until you’ve traversed that distance $0.8$ in arc length. This is the same thing as saying that this angle here is exactly $0.8$ radians, since the circle has radius $1$. Then $\sin(\theta)$ is your height above the $x$-axis at this point.
+
+<Figure
+  image="figures/13.16.svg"
+  video="figures/12.51-13.16.mp4"
+/>
+
+As theta increases, and you walk around the circle, your height bobs up and down and up and down, so the graph of $\sin(\theta)$ vs. $\theta$ is this wave pattern. The *quintessential* wave pattern.
+
+<Figure
+  image="figures/13.44.svg" 
+  video="figures/13.22-13.38.mp4"
+/>
+
+Now, just from looking at this graph, we can get a feel from the shape of the derivative function. The slope at $0$ is something positive, then as $\sin(\theta)$ approaches its peak, the slope goes down to $0$. Then the slope is negative for a little while before coming back up to $0$ as the $\sin(\theta)$ graph levels out. And, if you’re familiar with the graphs of trig functions, you might guess that this derivative graph should be exactly $\cos(\theta)$, whose graph is just a shifted-back copy of the sine graph.
+
+<Figure
+  image="figures/13.44-cosine.svg"
+  video="figures/13.40-14.30.mp4"
+  show="video"
+/>
+
+But all this tells us is that the peaks and valleys of the derivative graph seem to line up with the graph of cosine. How could we know that this derivative actually *is* the cosine of theta, and not just some new function that happens looks similar to it. Well, as with the previous examples of this video, a more exact understanding of the derivative requires looking at what the function itself represents, rather than just at the graph of the function.
+
+So think back to the walk around the unit circle, having traversed an arc length of theta, where $\sin(\theta)$ is the height of this point. Consider a slight nudge of $d$-theta along the circumference of the circle; a tiny step in your walk around the unit circle. How much does this change $\sin(\theta)$? How much does that step change your *height* above the x-axis? This is best observered by zooming in on the point where you are on the circle.
+
+<Figure
+  image="figures/15.57.svg" 
+/>
+
+Well, zoomed in close enough the circle basically looks like a straight line in this neighborhood, so let’s go ahead and think of this right here as a right triangle, where the hypotenuse represents that nudge of $d \theta$ along the circumference, and this left side represents the change in height; the resulting tiny nudge to $\sin(\theta)$.
+
+<Figure
+  image="figures/16.00.svg" 
+/>
+
+Now this tiny triangle is actually similar to this larger triangle with the defining angle theta, and whose hypotenuse is the radius of the circle with length $1$. Specifically, this little angle is precisely equal to $\theta$ radians. 
```

Comment:
```suggestion
This tiny triangle is actually similar to this larger triangle with the defining angle theta, and whose hypotenuse is the radius of the circle with length $1$.
Specifically, the angle between its height $d(\sin(\theta))$ and its hypotenuse $d\theta$ is precisely equals to $\theta$.
```

Reply:
Here it would be helpful to show a side-by-side of the two relevant triangles. We should also include an explanation (perhaps presented as a challenge question). It basically comes down to the tiniest bit of angle chasing.

![image](https://user-images.githubusercontent.com/11601040/124304968-4a2c1380-db19-11eb-90af-f195e2802cb1.png)